NitroxClient/MonoBehaviours/Gui/Input/KeyBindings/Actions/ChatKeyBindingAction.cs

```diff
@@ -1,17 +1,24 @@
-﻿using NitroxClient.GameLogic.ChatUI;
+using System;
+using NitroxClient.GameLogic.ChatUI;
 using NitroxModel.Core;
+using UnityEngine.InputSystem;
 
-namespace NitroxClient.MonoBehaviours.Gui.Input.KeyBindings.Actions
+namespace NitroxClient.MonoBehaviours.Gui.Input.KeyBindings.Actions;
+
+public class ChatKeyBindingAction : KeyBinding
 {
-    public class ChatKeyBindingAction : KeyBindingAction
+    private readonly Lazy<PlayerChatManager> playerChatManager = new(NitroxServiceLocator.LocateService<PlayerChatManager>);
```

Comment:
Couldn't that be injected by constructor ?

Right now the only purpose of KeyBindingManager is to host constant values

@Measurity

Reply:
Should I do the PlayerChatManager refactoring now ? Because ChatKeyBindingAction is manually initialized in a static manager so it doesn't know about DI (and it should definitely not since Keybindings are working at all times since game start). As of now, if you try opening the chat in the main menu, the lazy will register a null reference (no scope yet) and then it won't work in game.